meson.build

```diff
@@ -0,0 +1,83 @@
+project('euclidean-rhythms.lv2','c',version: '0.1.0-SNAPSHOT',default_options: ['default_library=shared','c_std=c99'])
+
+# Install folder
+lv2_directory = join_paths(get_option('libdir'), 'lv2')
+install_folder = join_paths(lv2_directory, meson.project_name())
+
+# Sources to compile
+euclidean_rhythms_src = ['plugins/eurhythms.c', 'src/euclidean_algorithms.c']
+
+# Dependencies for euclidean rhythms
+lv2_dep = dependency('lv2', required: true)
+m_dep = meson.get_compiler('c').find_library('m', required: true)
+all_dep = [lv2_dep,m_dep]
+
+# Get the host operating system and cpu architecture
+current_os = host_machine.system()
+current_arch = build_machine.cpu_family()
+
+# Shared c_args for libraries
+lib_c_args = ['-fvisibility=hidden']
```

Comment:
This is not necessary...

Reply:
Sorry to a ask but, why? The manpage for gcc says:

       -fvisibility=[default|internal|hidden|protected]
           Set the default ELF image symbol visibility to the specified option---all symbols are marked with
           this unless overridden within the code.  Using this feature can very substantially improve linking
           and load times of shared object libraries, produce more optimized code, provide near-perfect API
           export and prevent symbol clashes.  It is strongly recommended that you use this in any shared
           objects you distribute.

           Despite the nomenclature, default always means public; i.e., available to be linked against from
           outside the shared object.  protected and internal are pretty useless in real-world usage so the only
           other commonly used option is hidden.  The default if -fvisibility isn't specified is default, i.e.,
           make every symbol public.

Is not preferable to hide what doesn't need to be exposed?